packages/core/src/common/classes.ts

```diff
@@ -64,15 +71,31 @@ export const ELEVATION_4 = "pt-elevation-4";
 
 export const INPUT = "pt-input";
 export const INPUT_GROUP = "pt-input-group";
+export const CHECKBOX = "pt-checkbox";
+export const RADIO = "pt-radio";
+export const SWITCH = "pt-switch";
+export const FILE_UPLOAD = "pt-file-upload";
+export const FILE_UPLOAD_INPUT = "pt-file-upload-input";
+
+export const INTENT_PRIMARY = "pt-intent-primary";
+export const INTENT_SUCCESS = "pt-intent-success";
+export const INTENT_WARNING = "pt-intent-warning";
+export const INTENT_DANGER = "pt-intent-danger";
```

Comment:
We should probably moves these intent variables up in the modifiers section, right?